### PR TITLE
Applied isort 

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
This PR applies isort to all eox-tenant imports using:
`isort --recursive ./eox_tenant`
Following the config added to setup.cfg
This is done so it can be added this isort check to circle ci quality tests:
`isort --check-only --recursive --diff ./eox_tenant` 

@andrey-canon 
@morenol 
@felipemontoya

### CODE REVIEW

**Comment by morenol** [comment]
@mariajgrimaldi  Can you add a new commit adding that check to the quality tests?

File: eox_tenant/edxapp_wrapper/backends/edx_auth_i_v1.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 """ Backend abstraction. """
-from openedx.core.djangoapps.oauth_dispatch.dot_overrides.backends import EdxRateLimitedAllowAllUsersModelBackend  # pylint: disable=import-error
+# pylint: disable=import-error
```
Comment (andrey-canon) [inline eox_tenant/edxapp_wrapper/backends/edx_auth_i_v1.py:2]: With this change you are disabling all the import errors, and we don't want to do that, if this line is too long my suggestion is the following
``` 
from openedx.core.djangoapps.oauth_dispatch.dot_overrides.backends import (  # pylint: disable=import-error
    EdxRateLimitedAllowAllUsersModelBackend,
)
```

Comment (mariajgrimaldi) [reply eox_tenant/edxapp_wrapper/backends/edx_auth_i_v1.py:2]: Thank you! I'll change it

**Comment by mariajgrimaldi** [comment]
To circle ci or makefile? @morenol

**Comment by morenol** [comment]
Circle ci:

After these lines:

https://github.com/eduNEXT/eox-tenant/blob/0fa49c9f4d04fc381e88f69e7d12d74f3dbc4a38/.circleci/config.yml#L50

and 

https://github.com/eduNEXT/eox-tenant/blob/0fa49c9f4d04fc381e88f69e7d12d74f3dbc4a38/.circleci/config.yml#L87

**Comment by mariajgrimaldi** [comment]
All done! @morenol @andrey-canon

**Pull request merged**